### PR TITLE
Create command that navigates to the homepage and a command that goes up in the URL hiearchy.

### PR DESCRIPTION
**Pull request by pdelfino**
This solves half of issue #1643 with the creation of the  `go-to-homepage` command. 

I am still struggling with regular expressions `cl-ppcre` to build the other command (`go-up`). While I advance on the other, you guys can review this one.

Thanks.

### CODE REVIEW

File: source/web-mode.lisp [hunk 1]
```diff
@@ -298,6 +298,13 @@ This was useful before Nyxt 2.0 as a workaround for hangs that would occur on pa
                                  (query-selector-all "[rel=prev]") 0 (click))))
     (go-previous)))
 
+(define-command go-to-homepage ()
+  "Navigate to the homepage."
+  (let* ((url-address (url (current-buffer)))
+         (host (quri:uri-host url-address))
+         (scheme (quri:uri-scheme url-address)))
+    (buffer-load (concatenate 'string scheme "://" host))))
```
Comment (Ambrevar) [inline source/web-mode.lisp:306]: Don't use `concatenate` for string concatenation, prefer `str:concat`.
(There are other options, like `uiop:strcat`.)

File: source/web-mode.lisp [hunk 2]
```diff
@@ -298,6 +298,13 @@ This was useful before Nyxt 2.0 as a workaround for hangs that would occur on pa
                                  (query-selector-all "[rel=prev]") 0 (click))))
     (go-previous)))
 
+(define-command go-to-homepage ()
+  "Navigate to the homepage."
+  (let* ((url-address (url (current-buffer)))
+         (host (quri:uri-host url-address))
```
Comment (Ambrevar) [inline source/web-mode.lisp:304]: prefer `uri-authority` over `uri-host` since there may be credentials in the URI.

**Comment by Ambrevar** [comment]
You don't need regexp (in fact, avoid regexp as much as possible in general) to manipulate URI.

Instead, you should leverage the `quri` library.  It parses the URI for you, including the path.

**Comment by pdelfino** [comment]
Hi guys,

I think this is good to go!

**Pull request merged**